src/test/java/io/aiven/connect/elasticsearch/ElasticsearchSinkTestBase.java

```diff
@@ -138,38 +187,4 @@ protected void verifySearchResults(
             }
         }
     }
-
-    /* For ES 2.x */
-    @Override
-    protected Settings nodeSettings(final int nodeOrdinal) {
-        return Settings.settingsBuilder()
-            .put(super.nodeSettings(nodeOrdinal))
-            .put(IndexMetaData.SETTING_NUMBER_OF_SHARDS, 1)
-            .put(IndexMetaData.SETTING_NUMBER_OF_REPLICAS, 1)
-            .put(Node.HTTP_ENABLED, true)
-            .build();
-    }
-
-    /* For ES 5.x (requires Java 8) */
```

Comment:
why do we want to keep those snippets?

Reply:
sorry for the noise, my fault. This was in the previous version. Didn't notice I was reading the red portion of the diff 😅